notebooks/lesson14_image-dataset.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>should we add the usual step about upgrading the <code>dslectures</code>  library?</p><p> </p><pre class="ql-syntax" spellcheck="false"># uncomment if running locally or on Google Colab
# !pip install --upgrade dslectures
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>